.github/workflows/browser-tests.yml

```diff
@@ -6,7 +6,7 @@ on:
                 required: true
                 type: string
             project-version:
-                description: "Project version to set up: ^3.3.x-dev, ^4.0.x-dev etc."
+                description: "Project version to set up: ^3.3.x-dev, 4.0.x-dev etc."
```

Comment:
Why first has dash and the second doesn't? :)

Reply:
When creating stable branches for 4.1 and 4.2 I've noticed that Composer treats `^` in the `^4.1.x-dev` version contraint as in other constraints: minor bumps are allowed.

That's why `^4.1.x-dev` actually installs 4.2.x-dev:
```
Creating a "ibexa/website-skeleton:^4.1.x-dev" project at "./v3_4"
Info from https://repo.packagist.org: #StandWithUkraine
Installing ibexa/website-skeleton (4.2.x-dev d0d1eb89358e598a2adb1b2ae6aa2d7f23e70a3f)
```

To install 4.1.x-dev we should to use either `~4.1.x-dev` or `4.1.x-dev` AND it has to match a script present in ci-scripts: https://github.com/ibexa/ci-scripts/tree/main/bin

So:
- for v3 we must use `^3.3.x-dev` (there's no 3.4.x-dev, so it doesn't matter)
- for v4 we must use `4.0.x-dev`, `4.1.x-dev`, `4.2.x-dev` (without the `^`)

Reply:
I've expanded the description to include more examples for v4